javacord-core/src/main/java/org/javacord/core/util/gateway/AudioUdpSocket.java

```diff
@@ -79,12 +79,12 @@ public void setSecretKey(byte[] secretKey) {
      * @see <a href="https://discord.com/developers/docs/topics/voice-connections#ip-discovery">Discord Docs</a>
      */
     public InetSocketAddress discoverIp() throws IOException {
-        byte[] buffer = new byte[70];
+        byte[] buffer = new byte[74];
         ByteBuffer.wrap(buffer).putInt(0, ssrc);
```

Comment:
```suggestion
        ByteBuffer.wrap(buffer)
            .putShort((short) 1)
            .putShort((short) 70)
            .putInt(ssrc);
```

Ref: https://discord.com/developers/docs/topics/voice-connections#ip-discovery
Apparently this changed in 2019 and we never noticed